### PR TITLE
Completed wk 6 hooks / fragments assignments

### PR DESCRIPTION
**Pull request by poisonivy91**
This pull request includes:
- Implementation of custom hooks for persistent state management with localStorage.
- Refactored components using React Fragments.
- LocalStorage integration to save and load todos.

Please review and provide feedback. I’ll wait to merge this until after approval.

### CODE REVIEW

File: react-todo/src/App.jsx [hunk 1]
```diff
@@ -1,14 +1,34 @@
...
-    setTodoList([...todoList, newTodo]); // Add the new todo to the list
+    setTodoList([...todoList, newTodo]); 
   };
 
+  // useEffect(() => {
+  //   localStorage.setItem("savedTodoList", JSON.stringify(todoList));
+  // }, [todoList]);
+
   return (
     <div>
```
Comment (akhilwaghmare) [inline react-todo/src/App.jsx]: Per the assignment instructions, change this `div` into a fragment `<>`

File: react-todo/src/App.jsx [hunk 2]
```diff
@@ -1,14 +1,34 @@
...
-  const [todoList, setTodoList] = useState([]); // Start with an empty list
+  const [todoList, setTodoList] = useSemiPersistentState("savedTodoList", []);
+  
 
   const addTodo = (newTodo) => {
-    setTodoList([...todoList, newTodo]); // Add the new todo to the list
+    setTodoList([...todoList, newTodo]); 
   };
 
+  // useEffect(() => {
```
Comment (akhilwaghmare) [inline react-todo/src/App.jsx]: Nit: Remove unused commented code for cleanup

**Review by akhilwaghmare — APPROVED** [review]
Nice job setting up local storage for the todo list! Requesting a couple minor changes, then you are good to merge!

File: react-todo/src/App.jsx (hunk 2 above)
Comment (poisonivy91) [reply react-todo/src/App.jsx]: removed comment coode

**Pull request merged**